fastlane/Fastfile

```diff
@@ -1,29 +1,31 @@
 SUPPORTED_LOCALES = [
-  { glotpress: "ar", google_play: "ar",  promo_config: {}},
-  { glotpress: "de", google_play: "de-DE",  promo_config: {} },
+  { glotpress: "ar",    google_play: "ar",     promo_config: {} },
+  { glotpress: "de",    google_play: "de-DE",  promo_config: false },
   { glotpress: "en-gb", google_play: "en-US",  promo_config: {} },
-  { glotpress: "es", google_play: "es-ES",  promo_config: {} },
-  { glotpress: "fr", google_play: "fr-CA",  promo_config: false },
-  { glotpress: "fr", google_play: "fr-FR",  promo_config: {} },
-  { glotpress: "he", google_play: "iw-IL",  promo_config: {} },
-  { glotpress: "id", google_play: "in",  promo_config: {} },
-  { glotpress: "it", google_play: "it-IT",  promo_config: {} },
-  { glotpress: "ja", google_play: "ja-JP",  promo_config: {} },
-  { glotpress: "ko", google_play: "ko-KR",  promo_config: {} },
-  { glotpress: "nl", google_play: "nl-NL",  promo_config: {} },
-  { glotpress: "pl", google_play: "pl-PL",  promo_config: {} },
-  { glotpress: "pt-br", google_play: "pt-BR",  promo_config: {} },
-  { glotpress: "ru", google_play: "ru-RU",  promo_config: {} },
-  { glotpress: "sr", google_play:  "sr",  promo_config: {} },
-  { glotpress: "sv", google_play: "sv-SE",  promo_config: {} },
-  { glotpress: "th", google_play: "th",  promo_config: {} },
-  { glotpress: "tr", google_play: "tr-TR",  promo_config: {} },
-  { glotpress: "vi", google_play: "vi",  promo_config: {} },
-  { glotpress: "zh-cn", google_play: "zh-CN",  promo_config: {} },
-  { glotpress: "zh-tw", google_play: "zh-TW",  promo_config: {} },
+  { glotpress: "es",    google_play: "es-ES",  promo_config: false },
+  { glotpress: "fr",    google_play: "fr-CA",  promo_config: false },
+  { glotpress: "fr",    google_play: "fr-FR",  promo_config: false },
+  { glotpress: "he",    google_play: "iw-IL",  promo_config: false },
+  { glotpress: "id",    google_play: "in",     promo_config: false },
+  { glotpress: "it",    google_play: "it-IT",  promo_config: false },
+  { glotpress: "ja",    google_play: "ja-JP",  promo_config: false },
+  { glotpress: "ko",    google_play: "ko-KR",  promo_config: false },
+  { glotpress: "nl",    google_play: "nl-NL",  promo_config: false },
+  { glotpress: "pl",    google_play: "pl-PL",  promo_config: false },
+  { glotpress: "pt-br", google_play: "pt-BR",  promo_config: false },
+  { glotpress: "ru",    google_play: "ru-RU",  promo_config: false },
+  { glotpress: "sr",    google_play: "sr",     promo_config: false },
+  { glotpress: "sv",    google_play: "sv-SE",  promo_config: false },
+  { glotpress: "th",    google_play: "th",     promo_config: false },
+  { glotpress: "tr",    google_play: "tr-TR",  promo_config: false },
+  { glotpress: "vi",    google_play: "vi",     promo_config: false },
+  { glotpress: "zh-cn", google_play: "zh-CN",  promo_config: false },
+  { glotpress: "zh-tw", google_play: "zh-TW",  promo_config: false },
```

Comment:
This reduces the locales to only 2 due to my FTL quota.
🚩 Commit db7f192992b089abb62e646eb460c637558e3054 needs to be reverted to re-enable all locales.